src/main/java/org/bookkeepervisualmanager/config/ServerConfiguration.java

```diff
@@ -0,0 +1,60 @@
+/*
+ Licensed to Diennea S.r.l. under one
+ or more contributor license agreements. See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership. Diennea S.r.l. licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+
+ */
+package org.bookkeepervisualmanager.config;
+
+/**
+ *
+ * @author matteo.minardi
+ */
+public final class ServerConfiguration {
```

Comment:
We can use directly BookKeeper a Client Configuration object

Reply:
Should I call them in the same way?
I think they are alla package protected https://bookkeeper.apache.org/docs/latest/api/javadoc/org/apache/bookkeeper/conf/AbstractConfiguration.html#METADATA_SERVICE_URI